core/src/main/java/de/danoeh/antennapod/core/storage/DBUpgrader.java

```diff
@@ -326,6 +326,11 @@ static void upgrade(final SQLiteDatabase db, final int oldVersion, final int new
             db.execSQL("ALTER TABLE " + PodDBAdapter.TABLE_NAME_FEEDS
                     + " ADD COLUMN " + PodDBAdapter.KEY_MINIMAL_DURATION_FILTER + " INTEGER DEFAULT -1");
         }
+
+        if (oldVersion < 2090000) {
```

Comment:
Please use version `2060000` because 2.6.0 is the version this will first be released in (not 2.9.0)